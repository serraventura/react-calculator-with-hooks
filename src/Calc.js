import React from "react";

const Calc = () => {
  return;
};

export default Calc;
