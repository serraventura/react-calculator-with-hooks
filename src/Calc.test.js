import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Calc from "./Calc";

configure({ adapter: new Adapter() });

describe("<Calc />", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Calc />);
    expect(wrapper).toBeTruthy();
  });

  it("should render without crashing", () => {});

  it("should match snapshot", () => {});
});
