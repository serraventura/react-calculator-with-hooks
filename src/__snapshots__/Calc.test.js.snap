// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Calc /> should match snapshot 1`] = `ShallowWrapper {}`;
